tests/integration_tests/clouds.py

```diff
@@ -86,13 +87,25 @@ def _get_initial_image(self, **kwargs) -> str:
             CURRENT_RELEASE.series, **kwargs
         )
 
-    def _perform_launch(self, launch_kwargs, **kwargs) -> BaseInstance:
+    def _maybe_wait(self, pycloudlib_instance, wait):
+        if wait:
+            try:
+                pycloudlib_instance.wait()
+            except Exception:
+                pycloudlib_instance.delete()
+                raise
+
+    def _perform_launch(
+        self, *, wait=True, launch_kwargs, **kwargs
```

Comment:
This looks funny.

It's probably just me, but being used to keyword arguments with defaults being at the end of the function signature makes this look a little odd. It's fine if you leave it, I guess, since `launch_kwargs` is also a keyword argument.